content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -489,10 +420,32 @@ Serverless: - node_modules/@prisma/engines/prisma-fmt-darwin
 Serverless: Copying existing artifacts...
 ```
 
+### Remove unneeded artifacts
+
+If you do not use the Serverless Framework, you can use the following method to remove any unneeded artifacts.
```

Comment:
Good question. I assumed that it means files, but actually, our docs suggest it might mean files, binaries, or database elements! I'll ask in Slack and see what people think.

Reply:
In this context, it means "files", so I'll change it accordingly. We use the term in other ways in our docs, but that's a separate issue. In fact, I'll create a docs ticket to look into this.